WorkManagerSample/benchmark/src/androidTest/java/com/example/benchmark/WorkerBenchmark.kt

```diff
@@ -0,0 +1,123 @@
+package com.example.benchmark
+
+import android.content.Context
+import android.graphics.BitmapFactory
+import android.util.Log
+import androidx.arch.core.executor.testing.InstantTaskExecutorRule
+import androidx.benchmark.BenchmarkRule
+import androidx.test.InstrumentationRegistry
+import androidx.test.filters.SmallTest
+import androidx.test.runner.AndroidJUnit4
+import androidx.work.*
+import androidx.work.impl.utils.SynchronousExecutor
+import androidx.work.testing.WorkManagerTestInitHelper
+import com.example.lib.Constants
+import com.example.lib.workers.BaseFilterWorker
+import com.example.lib.workers.BaseFilterWorker.Companion.inputStreamFor
+import com.example.lib.workers.BlurEffectFilterWorker
+import com.example.lib.workers.GrayScaleFilterWorker
+import com.example.lib.workers.WaterColorFilterWorker
+import kotlinx.coroutines.runBlocking
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.runner.RunWith
+
+@RunWith(AndroidJUnit4::class)
+@SmallTest
+class WorkerBenchmark {
+
+    companion object {
+        private const val JETPACK = "${BaseFilterWorker.ASSET_PREFIX}images/jetpack.png"
+    }
+
+    private lateinit var mContext: Context
```

Comment:
don't use mFoo prefix in kotlin, only in .java

Reply:
:+1: